iree/compiler/Conversion/LinalgToSPIRV/ConvertToGPUPass.cpp

```diff
@@ -498,22 +520,47 @@ static LogicalResult mapToGlobalInvocationId(
       rewriter, pLoopOp);
 }
 
+/// Compute the number of bytes copied when load/storing to/from workgorup
+/// memory. It is appsoximated as the size of the underlying allocation being
```

Comment:
typo: appsoximated
Also, did you mean it is an approximation of the size of the underlying allocation?